datafusion-examples/examples/advanced_udwf.rs

```diff
@@ -75,6 +78,30 @@ impl WindowUDFImpl for SmoothItUdf {
         Ok(DataType::Float64)
     }
 
+    /// rewrite
+    fn simplify(
```

Comment:
This might be confusing as now this example UDWF will never call the actual implementation.

Perhaps we can make a second UDWF and use it as an example of wrapping an existing UDWF 🤔